docs/howtoguides/trusty_legacy_support.rst

```diff
@@ -0,0 +1,124 @@
+How to enable esm-infra-legacy on Trusty
+*****************************************
+
+The ``esm-infra-legacy`` service can be enabled with the Legacy Support add-on to grant an additional two years of 
+extended security coverage. This service is only available on 14.04 LTS (Trusty), since the release has reached
+the end of its support period for ``esm-infra``. Check out this article to `learn more about the expansion of Long Term Support for Trusty <https://canonical.com/blog/canonical-expands-long-term-support-to-12-years-starting-with-ubuntu-14-04-lts>`_ and how to contact Canonical to purchase this additional support.
+
+
+Make sure ``ua`` is up-to-date
+===============================
+
+All Trusty systems come with ``ua`` pre-installed through the
+``ubuntu-advantage-tools`` package. To make sure that you're running the latest
+version of ``ua``, run the following commands:
+
+.. code-block:: bash
+
+    sudo apt update && sudo apt install ubuntu-advantage-tools
+
+Note that on Trusty, Ubuntu Pro is referred to by its previous name: "Ubuntu Advantage" (or ``ua``).
+
+
+Check the status of the services
+================================
+
+After you have attached to a Pro subscription and
+updated the ``ubuntu-advantage-tools`` package, you can check which services
+are enabled by running the following command:
+
+.. code-block:: bash
+
+    sudo ua status
+
+This will show you which services are enabled or disabled on your machine,
+(output truncated for brevity). With the addition of Legacy Support, it will
+now show the esm-infra-legacy service on Trusty:
+
+.. code-block:: text
+
+    SERVICE          ENTITLED  STATUS    DESCRIPTION
+    esm-infra        yes       enabled   Expanded Security Maintenance for Infrastructure
+    esm-infra-legacy yes       disabled  Expanded Security Maintenance for Infrastructure on Legacy Instances
+    livepatch        yes       enabled   Canonical Livepatch service
+
+.. _enable_legacy:
+
+Enabling esm-infra-legacy
+==========================
+
+If you are entitled to the ``esm-infra-legacy`` service as shown above, you can enable it by running:
+
+.. code-block:: bash
+
+    sudo ua enable esm-infra-legacy
+
+
+What if ``esm-infra-legacy`` is not entitled?
+===================================================
+
+If ``ua status`` shows that you are not entitled to the service, you
+will first need to buy access to the service, as mentioned at the top of this page.
+
+Once you have purchased a support subscription, run the following command
```

Comment:
Suggest "Once you have purchased the Legacy Support add-on, run the following command"

Why? So users and customers are absolutely clear that this is not included in every subscription to Ubuntu Pro.